regression/tests/studio/test_studio_home.py

```diff
@@ -47,3 +51,56 @@ def test_studio_course_select(self):
         self.studio_home_page.visit()
         self.studio_home_page.select_course(get_course_display_name())
         self.studio_course_outline.wait_for_page()
+
+
+class StudioFooterTest(WebAppTest):
+    """
+    Tests for Studio Footer
+    """
+
+    DEMO_COURSE_USER = os.environ.get('USER_LOGIN_EMAIL')
+    DEMO_COURSE_PASSWORD = os.environ.get('USER_LOGIN_PASSWORD')
+
+    def setUp(self):
+        """
+        Initialize the page object
+        """
+        super(StudioFooterTest, self).setUp()
+
+        studio_login = StudioLoginApi()
+        studio_login.authenticate(self.browser)
+
+        self.terms_of_service = TermsOfService(self.browser)
+        self.privacy_policy = PrivacyPolicy(self.browser)
+        self.studio_home_page = DashboardPageExtended(self.browser)
+        self.course_info = get_course_info()
+        self.studio_course_outline = CourseOutlinePage(
+            self.browser, self.course_info['org'], self.course_info['number'],
+            self.course_info['run'])
+
+        self.studio_home_page.visit()
+
+    @skipIf(
+        STUDIO_BASE_URL != STUDIO_STAGE_BASE_URL,
+        "No link on sandbox"
+    )  # LT-62
+    def test_studio_footer_links(self):
+        """
+        Verifies that user can click and navigate to studio footer links
+        Terms of Service
+        Privacy Policy
+        """
+        self.studio_home_page.click_terms_of_service()
+        # Temporary condition for Acquia issue
+        self.assertTrue(
+            'Web Site Not Found' in self.browser.title or
+            self.terms_of_service.wait_for_page()
```

Comment:
I'm not certain, but I think the test might be more reliable to rely on wait_for_page, and move the `or` check inside `is_browser_on_page` for each page.